core/src/main/java/org/openstack4j/model/storage/block/builder/BlockQuotaSetBuilder.java

```diff
@@ -32,4 +32,6 @@
      * @return space consumed in the Block Storage.
      */
     BlockQuotaSetBuilder gigabytes(int gigabytes);
+
+    BlockQuotaSetBuilder volumeTypeQuota(String key, int quota);
```

Comment:
The builder API should accept whole map and not add entries one by one. It is more flexible as that support clearing and removal.